frontend/src/views/SubmissionView.vue

```diff
@@ -70,7 +76,11 @@ onMounted(async () => {
     />
     <span>Back to Submissions</span>
   </router-link>
-  <h1>Activity submission</h1>
+  <h1>
+    Activity submission:
+    <span v-if="submission?.confirmationId">{{ submission.confirmationId }}</span>
+    <span v-if="submission?.projectName">&nbsp;- {{ submission.projectName }}</span>
```

Comment:
CSS over HTML always. Replace &nbsp with padding

Reply:
Took out the CSS and put in a padding right in the first span. Auto formatter does something that adds an extra space if using padding left.